static/app/views/monitors/details.tsx

```diff
@@ -25,16 +27,25 @@ type State = AsyncView['state'] & {
 };
 
 class MonitorDetails extends AsyncView<Props, State> {
+  static contextTypes = {
+    router: PropTypes.object,
```

Comment:
This router context doesn't seem to be used anywhere.

Reply:
Yeah, it is used by AsyncView/AsyncComponent though.